Interop/OCI/src/main/java/org/tribuo/interop/oci/OCIModelCLI.java

```diff
@@ -0,0 +1,418 @@
+     */
+    private static void modelScoring(OCIModelOptions options) throws IOException, ClassNotFoundException {
+        // Load the dataset
+        Dataset<Label> dataset;
+        try (ObjectInputStream ois = new ObjectInputStream(Files.newInputStream(options.datasetPath))) {
+            dataset = Dataset.castDataset((Dataset<?>) ois.readObject(), Label.class);
+        }
+        ImmutableFeatureMap featureIDMap = dataset.getFeatureIDMap();
+
+        if (options.useOCIModel) {
+            // Prep mappings
+            Map<String, Integer> featureMapping = new HashMap<>();
+            for (VariableInfo f : featureIDMap){
+                VariableIDInfo id = (VariableIDInfo) f;
+                featureMapping.put(id.getName(),id.getID());
+            }
+            Map<Label, Integer> outputMapping = new HashMap<>();
+            for (Pair<Integer,Label> l : dataset.getOutputIDInfo()) {
+                outputMapping.put(l.getB(), l.getA());
+            }
+            OCIModel<Label> model = OCIModel.createOCIModel(dataset.getOutputFactory(),featureMapping,outputMapping,options.ociConfigFile,options.endpointDomain + options.modelDeploymentId, new OCILabelConverter(true));
+
+            System.out.println("Scoring using OCIModel - " + model.toString());
+            LabelEvaluator eval = new LabelEvaluator();
+            long startTime = System.currentTimeMillis();
+            LabelEvaluation evaluation = eval.evaluate(model,dataset);
+            long endTime = System.currentTimeMillis();
+            System.out.println("Scoring took - " + Util.formatDuration(startTime,endTime));
+            System.out.println((((double) endTime - startTime) / dataset.size()) + "ms per example");
+
+            System.out.println(evaluation.toString());
+            System.out.println(evaluation.getConfusionMatrix().toString());
+        } else {
+            manualModelScoring(options,dataset,featureIDMap);
+        }
+    }
+
+    private static void manualModelScoring(OCIModelOptions options, Dataset<Label> dataset, ImmutableFeatureMap featureIDMap) throws IOException {
+        OCILabelConverter labelConverter = new OCILabelConverter(true);
+
+        // Setup object mapper for parsing the output
+        ObjectMapper mapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+        FilterProvider filters =
+                new SimpleFilterProvider()
+                        .addFilter(ExplicitlySetFilter.NAME, ExplicitlySetFilter.INSTANCE);
+        mapper.setFilterProvider(filters);
+
+        // Pre-Requirement: Allow setting of restricted headers. This is required to allow the SigningFilter
+        // to set the host header that gets computed during signing of the request.
+        System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
+
+        // 1) Create a request signing filter instance
+        RequestSigningFilter requestSigningFilter = RequestSigningFilter.fromConfigFile(
```

Comment:
It could be quite annoying not to be able to specify your config file location and which profile to use

Reply:
Fixed.